srcpkgs/hplip-plugin/template

```diff
@@ -0,0 +1,61 @@
+# Template file for 'hplip-plugin'
+# Must be in sync with main hplip package
+pkgname=hplip-plugin
+version=3.22.10
+revision=1
+archs="i686 x86_64 armv[67]l aarch64"
+hostmakedepends="tar"
+short_desc="Binary plugin for HPs hplip printer driver library"
+maintainer="biopsin <biopsin@teknik.io>"
+license="custom:proprietary"
+homepage="https://developers.hp.com/hp-linux-imaging-and-printing/binary_plugin.html"
+changelog="https://developers.hp.com/hp-linux-imaging-and-printing/release_notes"
+distfiles="https://developers.hp.com/sites/default/files/hplip-${version}-plugin.run"
+checksum=bb9648ea0626a9b3cfa29e260348d5d617f328c55b66eda346384c04f4740b1f
+repository=nonfree
+restricted=yes
+
+ignore_elf_dirs="/usr/share/hplip"
+
+do_extract() {
+	/bin/sh ${XBPS_SRCDISTDIR}/${pkgname}-${version}/hplip-${version}-plugin.run \
```

Comment:
```suggestion
	/bin/sh ${wrksrc}/hplip-${version}-plugin.run \
```
or even this if it's possible
```suggestion
	/bin/sh ./hplip-${version}-plugin.run \
```

Reply:
I've tested both of these suggestions but neither worked.

Because of this I've not changed this with the latest push.